docs/src/appendix/chisel-plugin.md

```diff
@@ -0,0 +1,76 @@
+# Notes on the Compiler Plug-in
+
+The Chisel plugin provides some operations that are too difficult, or not possbile, 
+to implement through regular Scala code.
+
+## Compiler plugin operations
+1. Automatically generates the `cloneType` methods of Bundles
+2. Records that this plugin ran on a bundle by adding a method `_usingPlugin: Boolean = true`
+3. Generates a function that returns list of the hardware fields in a bundle `_elementsImpl`
+
+### Generating `cloneType` method
+After a period of testing this is now on by default. This flag was used to control that prior to full adoption.
```

Comment:
```suggestion
After a period of testing this is now on by default (as of Chisel Foo.Baz.Biddle). This flag was used to control that prior to full adoption.
```

Reply:
Referenced date and pr where current behavior was implemented